bvt.py

```diff
@@ -59,8 +59,10 @@ def parse_args_or_exit(argv=None):
 
 if __name__ == "__main__":
     args = parse_args_or_exit(sys.argv[1:])
+    build_status = is_build_action_ok("trunk-ring3", "xe-phase-1-build")
     bvt_status = is_build_action_ok("trunk-ring3", "xe-phase-1-test-ring3")
     if args.dry_run:
```

Comment:
Can we add to dry-run? I often run with this for debug :) Also, did you change the permissions?